src/client/reduxStore/moviesReducer.js

```diff
@@ -0,0 +1,55 @@
+import * as types from './actionTypes';
+
+let initialState = {
+  searchText: '',
+  searchBy: 'title',
+  sortBy: 'release_date',
+  offset: 0,
+  total: 0,
+  isLoading: false,
+  movies: []
+}
+
+export function moviesReducer(state = initialState, action) {
```

Comment:
please refactor this in order that action have only two props: type and payload

Reply:
Refctored